addons/payment_adyen/models/payment_transaction.py

```diff
@@ -0,0 +1,200 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import pprint
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+import odoo.addons.payment.utils as payment_utils
+from odoo.addons.payment_adyen.models.payment_acquirer import CURRENCY_DECIMALS
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentTransaction(models.Model):
+
+    _inherit = 'payment.transaction'
+
+    adyen_payment_data = fields.Char(
```

Comment:
Everyone has a read access right on transactions, shouldn't we add a group on this ? 

I mean: public users have the read access right to transactions if 
```
    <record id="payment_transaction_user_rule" model="ir.rule">
        <field name="name">Access own payment transaction only</field>
        <field name="model_id" ref="payment.model_payment_transaction"/>
        <field name="domain_force">['|',('partner_id','=',False), ('partner_id','=',user.partner_id.id) ]</field>
        <field name="groups" eval="[(4, ref('base.group_user')), (4, ref('base.group_portal')), (4, ref('base.group_public'))]"/>
    </record>
```

It means that a public user could read payment data from any other public user transactions right ?

Reply:
Right, good catch.